### PR TITLE
put on pants for the first time in his life

### PR DESCRIPTION
**Pull request by segerpeter07**
Hey guys,

I've been working on this feature for a while.

I know that they fit, but I haven't tested if they fit other people.

### CODE REVIEW

File: pants.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+these are some really good pants
+
+they fit
```
Comment (phuston) [inline pants.txt]: Do they really?

File: pants.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+they feel good
```
Comment (phuston) [inline pants.txt]: That's good for you

File: pants.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+all is good
+
+
+
+
+
+peter seger, 2017
```
Comment (phuston) [inline pants.txt]: Looks good! Feel to merge, I'm okay believing you that the pants actually fit

File: pants.txt (hunk 1 above)
Comment (segerpeter07) [reply pants.txt]: Yes sir! All good to go on my side!

**Pull request merged**